R/check_predictions.R

```diff
@@ -161,10 +161,37 @@ check_predictions.stanreg <- function(object,
   )
 
   if (inherits(object, "brmsfit")) {
-    out <- bayesplot::pp_check(object, type = type, ndraws = iterations, ...)
+    out <- as.data.frame(bayesplot::pp_check(object, type = type, ndraws = iterations, ...))
```

Comment:
do we even need to call `bayesplot`? Couldn't we directly compute predictions like in https://github.com/easystats/performance/issues/202#issuecomment-783842658

though it might be less efficient

Reply:
Not sure how we can get dot-plots for discrete/integer outcomes?